### PR TITLE
vmwh: new package at ada51fe

### PR DESCRIPTION
**Pull request by cornett**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package
vmware userland helper:  https://github.com/jcs/vmwh

This package installs a single binary, vmwh, that allows cutting and pasting between vmware linux guests and hosts.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/vmwh/version
/var/db/kiss/installed/vmwh/sources
/var/db/kiss/installed/vmwh/patches/stdint.patch
/var/db/kiss/installed/vmwh/patches/remove-vis-dep.patch
/var/db/kiss/installed/vmwh/patches/packed-attr.patch
/var/db/kiss/installed/vmwh/patches/makefile.patch
/var/db/kiss/installed/vmwh/patches/
/var/db/kiss/installed/vmwh/manifest
/var/db/kiss/installed/vmwh/depends
/var/db/kiss/installed/vmwh/checksums
/var/db/kiss/installed/vmwh/build
/var/db/kiss/installed/vmwh/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/vmwh
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/vmwh/build [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+patch -p1 < makefile.patch
+
+# Include stdint.h to fix compile error.
+patch -p1 < stdint.patch
+
+# Add missing __packed macro.
+patch -p1 < packed-attr.patch
+
+# Remove debugging code that needs bsd strnvis(3).
+patch -p1 < remove-vis-dep.patch
```
Comment (dylanaraps) [inline community/vmwh/build]:
```sh
for patch in *.patch; do
    patch -p1 < "$patch"
done
```

File: community/vmwh/version [hunk 2]
```diff
@@ -0,0 +1 @@
+ada51fe 1
```
Comment (dylanaraps) [inline community/vmwh/version]: Version should be `git`.

File: community/vmwh/sources [hunk 3]
```diff
@@ -0,0 +1,5 @@
+git+https://github.com/jcs/vmwh#ada51feced4b4f203ec725264b893dee9c8930f7
```
Comment (dylanaraps) [inline community/vmwh/sources]: Drop the `#commit` portion. This software is likely to never get another commit.

File: community/vmwh/patches/makefile.patch [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+ $(PROG): $(OBJS)
+ 	$(CC) $(OBJS) $(LDPATH) $(LIBS) -o $@
+ 
+-$(OBJS): *.o: *.c
++$(OBJS): vmwh.h
++
++.c.o:
+ 	$(CC) $(CFLAGS) $(INCLUDES) -c $< -o $@
+ 
+ install: all
```
Comment (dylanaraps) [inline community/vmwh/patches/makefile.patch]: Does it refuse to build without this patch?

File: community/vmwh/build (hunk 1 above)
Comment (cornett) [reply community/vmwh/build]: I just amended the pull request to include this.

File: community/vmwh/version (hunk 2 above)
Comment (cornett) [reply community/vmwh/version]: Done, thanks.

File: community/vmwh/sources (hunk 3 above)
Comment (cornett) [reply community/vmwh/sources]: Done.

File: community/vmwh/patches/makefile.patch (hunk 4 above)
Comment (cornett) [reply community/vmwh/patches/makefile.patch]: No, it won't build with gnu make.  I did simplify the patch just a bit though.

**Comment by dylanaraps** [comment]
Looks good now. :+1:

**Pull request merged**